hyperjumptech/monika
# Fix crash when stun server is inaccessible

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Closing issue 564
2. WHen STUN server is inaccesible, monika crashes

## How did you implement / how did you fix it  
1.  Add ping package
2. Before sending request to STUN server,  ping it first.
3. Bypasses stun checking if connectin.allive returns false.

## How to test  
1. run monika normally, `monika -c monika.yml`
2. Ensure you get ip and location
3. Disable the network and ensure monika does not crash
